Dockerfile.test

```diff
@@ -14,9 +14,14 @@ COPY tests/requirements.txt ${HOME}/tests/
 RUN pip3 install -r requirements.txt -r tests/requirements.txt && \
     pip2 install wheel
 
-COPY . ${HOME}
+COPY Makefile ${HOME}/
+
+COPY ./tests/ ${HOME}/tests/
 RUN chown -R 1000 ${HOME}
 
+COPY . /tmp/tmp/
```

Comment:
eiwwwww

Reply:
Would /tmp/temp-xyz/ be better?
Seriously, what wrong with the above, given that we delete it on next line.

Reply:
I wonder if it really stays in the image since /tmp is usually tmpfs but afaik docker does not enforce it; leave it as it is